src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -22,6 +19,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
     End Class
 
+    Partial Friend Class BoundAddressOfOperator
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of BoundNode)
+            Get
+                Return ImmutableArray.Create(Of BoundNode)(Me.MethodGroup)
+            End Get
+        End Property
+    End Class
+
+    Partial Friend Class BoundMethodGroup
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of BoundNode)
+            Get
+                Dim builder = ImmutableArray.CreateBuilder(Of BoundNode)
+                If Me.ReceiverOpt IsNot Nothing Then
+                    builder.Add(Me.ReceiverOpt)
+                End If
+
+                If Me.TypeArgumentsOpt IsNot Nothing Then
+                    builder.Add(Me.TypeArgumentsOpt)
```

Comment:
>builder.Add(Me.TypeArgumentsOpt) [](start = 20, length = 32)

I believe we shouldn't add this node to the children list. We don't expose type arguments as IOperation nodes, they are either exposed through a symbol, or as an array of type symbols (for dynamic invocations). I suggest removing this code and moving implementation of Children property to BoundPropertyOrMethodGroup.